junit5/src/test/java/org/jboss/weld/junit5/explicitInjection/ExplicitParameterInjectionNestedClass2Test.java

```diff
@@ -0,0 +1,98 @@
+package org.jboss.weld.junit5.explicitInjection;
+
+import jakarta.enterprise.inject.Default;
+
+import org.jboss.weld.junit5.ExplicitParamInjection;
+import org.jboss.weld.junit5.WeldJunit5Extension;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(WeldJunit5Extension.class)
+@ExplicitParamInjection(false)
+public class ExplicitParameterInjectionNestedClass2Test {
+
+    @Test
+    public void testParameterResolution(@Default Foo foo, Bar bar, @MyQualifier BeanWithQualifier bean) {
+        // Bar should be resolved by another extension
```

Comment:
In this test, Bar is also resolved by Weld, or?

Reply:
Yes, the comment below is a misleading a copy-paste error meant to confuse any reviews :D